CMS-Enterprise/easi-app
# [NOREF] Add LCID and Expiration Date to portfolio report CSV

# NOREF

## Description

Adds `lcid` and `lcidExpiresAt` fields to portfolio report CSV

## How to test this change

- `scripts/dev up`
- `scripts/dev db:seed`
- Log in as an admin user
- Run portfolio export report targeting the -30 days to +30 days as a date range
- Ensure fields show up properly

> [!NOTE]
> LCIDs sometimes appear formatted as numbers (i.e. leading `0`'s stripped), but this is often just your CSV viewing program doing this. Open the file in a text editor to see the true values being output!

> [!NOTE]
> Want to see an example of an output from the testing instructions above?
> [EASi-Portfolio-Update-Report.csv](https://github.com/user-attachments/files/17512816/EASi-Portfolio-Update-Report.csv)

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
